README.md

```diff
@@ -124,7 +139,64 @@ mvn test -Dtest=SanityTests -X
         
   </details>
 
-<br/><br/>
+</br>
+
+> If you weren't successful at setting up you local env, then use the [Gitpod approach](#gitpod-setup)👇
+
+
+### Gitpod setup
+
+Open forked repo in Gitpod.io
+
+1. Sign up for a free [Github account](https://github.com/)
+2. Fork this repository
+    * Make sure you are logged into Github
+    * Click the fork in the upper right of Github
+    * Select your username as the location to fork the repo
+3. In the browser address bar, prepend the Github url (`https://github.com/<username>/automation-best-practices-java`) with `https://gitpod.io/#`
+    * The resulting url should look as follows:
+   
+      > gitpod.io/#https://github.com/{USERNAME}/automation-best-practices-java
+   
+4. Once the Gitpod.io URL is loaded, you will need to sign in with the Github account you created earlier 
+5. Once the development environment is loaded and you see 'Ready to test!', run the following commands to set your `SAUCE_USERNAME` and `SAUCE_ACCESS_KEY`:
```

Comment:
@c3ccl3ston I didn't see "ready to test". Instead I see what's in the image. However, I didn't fork the repo and only opened it directly...
![Screen Shot 2021-11-09 at 2 22 08 PM](https://user-images.githubusercontent.com/2136245/140990754-5ae2dee6-08b1-4875-947b-579fce42285c.png)